admin-dev/themes/default/js/bundle/product/default-category.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Default category management
  */
-const defaultCategory = (function () {
+window.defaultCategory = (function () {
```

Comment:
This modif should be removed from this PR before merge. (It just help testing this PR easily and seperately)